unify syntax in readmes

petrparolek opened the pull request.

petrparolek committed:
unify syntax in readmes

paveljanda commented:
Is there any indentation  convention when using neon files?

I personally use tabs. Therefor I would prefer using tabs also in here.

petrparolek committed:
unify syntax in readmes
- use 4 spaces in CSS, in JS, in HTML and in Latte
- use tab in NEON and in PHP
- neon highlighting syntax
- normalized filename of Readmes
- typos

paveljanda commented on .docs/README.md:
```diff
@@ -19,39 +19,39 @@ composer require contributte/jsonrpc
+        <a href="https://github.com/gameeapp">Gameeapp</a>
+      </td>
+    </tr>
+  </tbody>
```
Would it be pls possible to keep the tabs? I do not use spaces in html at all.. thx!

petrparolek replied:
Promin, nebudu pořád dokola hodiny synchronizovat přes 130 repositářů!

f3l1x replied:
Just keep tabs @petrparolek.

paveljanda replied:
You know what - I am probably older so I should step back.. I didn't notice it is only the maintainers block. 👍
